crates/hdk/src/lib.rs

```diff
@@ -128,17 +122,17 @@
 //! Most externs are simply available to external processes and must be called explicitly e.g. via RPC over websockets.
 //! The external process only needs to ensure the input and output data is handled correctly as messagepack.
 //!
-//! ## Internal callbacks
+//! ## Host functions = internal callbacks
```

Comment:
@jost-s what do you mean by this?

callbacks are the host calling the guest, such as `validate`

host functions are the functions provided by the host to the guest, e.g. the ones the HDK uses under the hood

Reply:
Yes, I see. So are the internal callbacks "Host callbacks"? These callbacks are host functions, aren't they?

Reply:
@thedavidmeister I get it now. The internal callbacks defined like extern functions in the guest, have reserved names, and are called by the host (also just like extern callbacks) if they are defined. So I left it at "internal callbacks" while I don't come up with a better term.